common/v2/features/BuyAndExchange/ShapeShift/ShapeShiftDisabled.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+// Legacy
+import TabSection from 'containers/TabSection';
+import Warning from 'components/ui/Warning';
+
+export default class ShapeShiftDisabled extends Component {
+  public render() {
+    return (
+      <TabSection>
+          <Warning>
+            ShapeShift functionality is currently unavailable on the Legacy MyCrypto App.
```

Comment:
Shouldn't we keep 'Legacy' as an internal reference and just refer to 'MyCrypto' or 'Mycrypto App' as whatever is on mycrypto.com

Reply:
That is a good point! I'll make the change

Reply:
Removed in 43e5320